docs/by-example/client.rst

```diff
@@ -139,6 +139,24 @@ Refresh locations:
 
     >>> cursor.execute("REFRESH TABLE locations")
 
+Updating Data
+=============
+
+Both when inserting or updating data, values for ``TIMESTAMP`` columns can be obtained
+in different formats. Both scalar strings and datetime objects are supported.
+
+    >>> import datetime as dt
+    >>> timestamp_full = "2023-06-26T09:24:00.123+02:00"
+    >>> timestamp_date = "2023-06-26"
```

Comment:
@matriv: Those examples, values of `timestamp_full` and `timestamp_date` types, would represent the "scalar strings" variant I was referring to. When referring to scalar vs. container/aggregate types, I usually follow this guidance. I recognize there may be nuances to it.

> Generally, all basic primitive data types are considered scalar.
>
> -- https://stackoverflow.com/a/10212398